packages/cluster-ui/src/statementsPage/statementsPage.tsx

```diff
@@ -227,6 +228,16 @@ export class StatementsPage extends React.Component<
       );
   };
 
+  calculateTotalWorkload = (statements: AggregateStatistics[]) => {
+    let totalWorkload = 0;
```

Comment:
You could drop this variable by using a `reduce` here instead of a `forEach`,

```jsx
  calculateTotalWorkload = (statements: AggregateStatistics[]) =>
    statements.reduce(
      (totalWorkload, stmt) =>
        (totalWorkload +=
          longToInt(stmt.stats.count) * stmt.stats.service_lat.mean),
      0,
    );

```